content/en/docs/about/community.md

```diff
@@ -30,40 +28,56 @@ The Kubeflow Slack workspace has many channels, here are a few examples:
 | Users - Korea | [#users-korea](https://kubeflow.slack.com/archives/CKPCJB9AP)
 | Users - Oceania | [#users-oceania](https://kubeflow.slack.com/archives/C023ZN1R9FC)
 
-## Mailing List
+## Kubeflow Mailing List
 
 The official Kubeflow mailing list is a Google Group called [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss).
 
-More detail about the Kubeflow mailing lists:
+<a href="https://groups.google.com/g/kubeflow-discuss">
+  <button class="btn btn-primary py-2 px-5">Click to join:<br>Kubeflow Mailing List</button>
+</a>
 
-| Topic | Mailing List |
-| --- | --- |
-| General Discussion | [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss)
+## Kubeflow Community Call
 
-## Weekly Community Call
+The Kubeflow community holds weekly Zoom calls for all users and contributors to discuss issues/proposals and present demos/products.
+Kubeflow community calls are held each Tuesday and alternate between `8:00am San Francisco` time and `8:00am London` time.
 
-The Kubeflow community holds a public call every Tuesday, alternating between `US East/EMEA` and `US West/APAC` friendly times.
+Joining the [mailing list](#kubeflow-mailing-list) should automatically add these events to your Google Calendar.
+Alternatively, you can manually subscribe to the ["Kubeflow Community" calendar](#kubeflow-community-calendars).
 
-{{% alert title="Tip" color="info" %}}
-Joining the [kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss) Google Group will automatically send a calendar invitation to your email address.
-{{% /alert %}}
+| Useful Links |
+| --- | --- |
+| Kubeflow Community Call - Notes | [Google Doc](http://bit.ly/kf-meeting-notes)
+| Kubeflow Community Call - Recordings | [YouTube Playlist](https://www.youtube.com/playlist?list=PLmzRWLV1CK_ypvsQu10SGRmhf2S7mbYL5)
+
+## Kubeflow Community Calendars
 
-More detail about the Kubeflow weekly community call:
+The following calendars are maintained by [Kubeflow Working Groups](#kubeflow-working-groups) and contain meetings that you may wish to attend.
 
-| Description | Link |
+| Calendar | Link |
```

Comment:
Can you add Pipelines to the list of Calendars? Thanks!

Reply:
@james-wu can the pipelines calendar actually is the "Kubeflow community" one.

Reply:
@james-jwu the pipelines calendar actually is the "Kubeflow community" one, so it's already included.

Possibly, I could include a row that says pipelines wg uses the community one, but that's probably not necessary?

Reply:
I see that Training is probably also under Kubeflow Community calendar. This is fine.